lib/v2/bilibili/watchlater.js

```diff
@@ -0,0 +1,44 @@
+const got = require('@/utils/got');
+const cache = require('./cache');
+const config = require('@/config').value;
+const utils = require('./utils');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const uid = String(ctx.params.uid);
```

Comment:
```suggestion
    const uid = ctx.params.uid;
```
The type will be either a string or `undefined`. No need to type cast it again.